solr/core/src/test/org/apache/solr/schema/TestCloudSchemaless.java

```diff
@@ -63,7 +63,8 @@ protected String getCloudSolrConfig() {
 
   @Override
   public SortedMap<ServletHolder, String> getExtraServlets() {
-    final SortedMap<ServletHolder, String> extraServlets = new TreeMap<>();
```

Comment:
I think I'd prefer how it was before, but that's subjective. Separately (and out of the scope of this pull request) wondering if the `getExtraServlets` stuff within the code base is still needed or if it can be removed perhaps.

Reply:
I took the method out and it didn't matter, can we consider it a "redundent method" and include it in the PR ;-)??

Reply:
Hmm, it perhaps didn't matter because it used the method from the base class? There's a little bit of `getExtraServlets` sprinkled across the code base -- https://github.com/apache/solr/search?q=getExtraServlets -- let's out-scope here and clean that up separately.

https://github.com/apache/solr/pull/924/commits/392f09ee184d61e92c8405d25ac0c25052b734d1 added to defer.